app/controllers/ChargeSummaryController.scala

```diff
@@ -197,15 +196,16 @@ class ChargeSummaryController @Inject()(val authenticate: AuthenticationPredicat
     val viewSection3 = isEnabled(ChargeHistory) && (isEnabled(CodingOut) && documentDetailWithDueDate.documentDetail.isPayeSelfAssessment)
 
     val values = List(
-      (viewSection1, documentDetailWithDueDate.documentDetail.originalAmount.isDefined, "Original Amount"),
+      (viewSection1, documentDetailWithDueDate.documentDetail.originalAmount, "Original Amount"),
```

Comment:
this change looks a bit strange, as .isDefined would return Boolean, but .originalAmount would return BigDecimal, 
my assumption is that we can just put "true" instead, as this field will be always with the value

Reply:
I put "true" for a similar case at line 202